src/base/static/components/molecules/activity-item.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "react-emotion";
+
+import { UserAvatar } from "../atoms/imagery";
+import { SmallText, Link } from "../atoms/typography";
+
+const ActivityLink = styled("a")(props => ({
+  textDecoration: "none",
+  fontFamily: props.theme.text.bodyFontFamily,
+  lineHeight: "1.1em",
+}));
+
+const BasicActivityItem = props => (
+  <ActivityLink href={props.url} rel="internal">
+    <li className={props.className}>
```

Comment:
Another case where Emotion passes the className prop through from `ActivityLink`...

Reply:
See my earlier comment about using a wrapper component like ActivityItemContainer